docs/src/main/resources/templates/scaling.html

```diff
@@ -0,0 +1,115 @@
+<h4 id="scaling_intro">
+  Introduction
+  <a class="headerlink" href="#scaling_intro" title="Permalink to this headline">¶</a>
+</h4>
+<p>
+  The print has a state do store the job queue and the job state, then we should store this start to be able
```

Comment:
do -> to

I don't understand "then we should store this start" and how this relates to horizontal scaling.

Reply:
start -> state

Reply:
OK so my proposal
"  The print has a state do store the job queue and the job state, then we should store this start to be able..."
->
 "The print has a state contains the job queue and each job's state. To enable horizontal scaling, these states must be persisted in a database."